workflow/github.md

```diff
@@ -78,12 +78,12 @@ If we haven't written a styleguide for the language you're working in, please fe
 
 Here are the steps that need to be completed before merging:
 
+- If your proposal is still incomplete, you should at first create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
```

Comment:
Maybe it's good to reiterate the point that if a PR has been created in good faith, but subsequently becomes "un-merge-worthy" you should use the appropriate labels. It's not currently possible to convert a regular PR to a draft.

Reply:
Yeah draft PRs don't work terribly well with my workflow. I'm not sure they should be an official recommendation. I think they're one of the 3 options:

- Draft PR - for when you deliberately create a PR before it's ready
- "WIP: " - for when you've created a non-draft PR, but realise you need to do more work
- "Don't merge" label - can be applied to draft or WIP PRs, but is also useful for PRs with finished work that simply can't be merged yet

Reply:
Ok, removed this line entirely.